Resources/Textures/_Harmony/Structures/Decoration/banner.rsi/meta.json

```
@@ -0,0 +1,15 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Sprite taken from https://github.com/space-wizards/space-station-14/pull/7253/commits/ae8eb3582677566bb2ffbc66695461ae617cd722 and modified by whythelettuce (Github).",
+    "size": {
+      "x": 32,
+      "y": 32
+    },
+    "states": [
+      {
+        "name": "banner_cargo"
+      }
+    ]
+  }
+  
```

Indentation should be 4 spaces, also your attribution should point to a commit or blob, not a PR. Also also, there should be an empty line at the end of every file (EOF). 

```suggestion
{
    "version": 1,
    "license": "CC-BY-SA-3.0",
    "copyright": "Sprite taken from https://github.com/space-wizards/space-station-14/commit/ae8eb3582677566bb2ffbc66695461ae617cd722. Modified by whythelettuce (Github).",
    "size": {
        "x": 32,
        "y": 32
    },
    "states": [
        {
            "name": "banner_cargo"
        }
    ]
}
```